test/test_browser.py

```diff
@@ -1353,6 +1353,32 @@ def test_fs_idbfs_sync(self, args):
     print('done first half')
     self.btest_exit('fs/test_idbfs_sync.c', cflags=['-lidbfs.js', f'-DSECRET="{secret}"', '-sEXPORTED_FUNCTIONS=_main,_test,_finish', '-lidbfs.js'] + args)
 
+  def test_fs_idbfs_autopersist(self):
+    common_cflags = ['-lidbfs.js', '-sEXPORTED_FUNCTIONS=_main,_test,_finish']
+    test_cases = [
+      'open',
+      'close',
+      'symlink',
+      'unlink',
+      'rename',
+      'mkdir',
+    ]
```

Comment:
Instead of this list here you can use `parameterize` to have the `case` case passed in as an argument.

Then each case gets its own test that can be run and debugged on its own.